package.json

```diff
@@ -52,6 +52,7 @@
     "build-travis": "cross-env NODE_ENV=production node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js --config webpack.prod.config.js",
     "heroku-postbuild": "npm run build",
     "lint": "tslint --project .",
+    "lint.andFix": "tslint --project . --fix",
```

Comment:
not sure if this `npm run lint.andFix` is easier to remember then `npm run lint -- --fix` :-)

Reply:
learning the "-- --param" syntax is far from the first thing I learned/memorized about running NPM NPM scripts.